pymc3/tests/test_broadcasting.py

```diff
@@ -0,0 +1,25 @@
+"""Tests for broadcasting of model variables."""
```

Comment:
I was about to suggest putting these test into an existing tests file, but it looks like there wasn't any test for `pm.make_shared_replacements` to begin with 😬

Reply:
I felt the same but didn't find anything 🤷‍♂️ 
Wasn't sure where it would otherwise fit...

Reply:
Why not in test_aesaraf, since this function is implemented there?